flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/datastream/DataStreamRel.scala

```diff
@@ -22,17 +22,16 @@ import org.apache.calcite.rel.RelNode
 import org.apache.flink.streaming.api.datastream.DataStream
 import org.apache.flink.table.api.StreamTableEnvironment
 import org.apache.flink.table.plan.nodes.FlinkRel
-import org.apache.flink.types.Row
 
-trait DataStreamRel extends RelNode with FlinkRel {
+trait DataStreamRel[T] extends RelNode with FlinkRel {
```

Comment:
I don't think we need to parameterize `DataStreamRel`. All `DataStreamRel` nodes should return `CRow` in my opinion.